features/db-search.feature

```diff
@@ -1066,3 +1066,27 @@ Feature: Search through the database
       :aöXYXYX
       """
     And STDERR should be empty
+
+  Scenario: Search for a string and output the format as a table
+    Given a WP install
+
+    When I run `wp db search example.com --format=csv`
+    Then STDOUT should contain:
+      """
+      wp_options,option_value,option_id,14,mail.example.com
+      wp_options,option_value,option_id,15,login@example.com
+      """
+
+    When I run `wp db search example.com --format=ids`
```

Comment:
```suggestion
    When I try `wp db search example.com --format=ids`
```

WHen you expect a failure, you need to use "try"

Reply:
OK, didn't know that. Thanks!